bin/fkCli.py

```diff
@@ -98,18 +108,28 @@ def process_cmd(client: FaaSKeeperClient, cmd: str, args: List[str]):
     function = getattr(client, clientAPIMapping[cmd])
     sig = signature(function)
     params_count = len(sig.parameters)
-    # incorrect number of parameters
-    if params_count != len(args):
-        msg = f"{cmd} arguments:"
-        for param in sig.parameters.values():
-            # "watch" requires conversion - API uses a callback
-            # the CLI is a boolean switch if callback should be use or not
-            if param.name == "watch":
-                msg += f" watch:bool"
-            else:
-                msg += f" {param.name}:{param.annotation.__name__}"
-        click.echo(msg)
-        return client.session_status, client.session_id
+    # check incorrect number of parameters
+    if cmd in CMD: # check parsed arguments
+        # in this case, number of parsed arguments is always correct, so we need to check it type
+        # if an argument is missing, it will be a boolean value
+        for idx, param in enumerate(sig.parameters.values()):
+            if param.annotation != bool and isinstance(args[idx], bool) and param.name != "watch":
+                click.echo(f"Command Example: {INPUT_FORMAT[cmd]}")
+                return client.session_status, client.session_id
+            if isinstance(args[idx], bool): # convert boolean to string
+                args[idx] = str(args[idx])
```

Comment:
What is the reason for converting to string? is it because of line 148?

Reply:
and line 140

Reply:
> and line 140

Ya. Great to see the latest commit, does look cleaner. I will test each command ASAP.